docs/tutorials/multimodal/multimodal_prediction/multimodel_ner.md

```diff
@@ -0,0 +1,138 @@
+# AutoMM for Multimodal Named Entity Extraction
+
+We have introduced how to train an entity extraction model with text data.
+Here, we move a step further by integrating data of other modalities.
+In many real-world applications, textual data usually comes with data of other modalities.
+For example, Twitter allows you to compose tweets with text, photos, videos, and GIFs. Amazon.com uses text, images, and videos to describe their products.
+These auxiliary modalities can be leveraged as additional context resolution of entities.
+Now, with AutoMM, you can easily exploit multimodal data to enhance entity extraction without worrying about the details.
+
+```{.python .input}
+import os
+import pandas as pd
+import warnings
+warnings.filterwarnings('ignore')
+```
+
+## Get the Twitter Dataset
+In the following example, we will demonstrate how to build a multimodal named entity recognition model with a real-world [Twitter dataset](https://github.com/jefferyYu/UMT/tree/master).
+This dataset consists of scrapped tweets from 2016 to 2017, and each tweet was composed of one sentence and one image. Let's download the dataset.
+
+```{.python .input}
+download_dir = './ag_automm_tutorial_ner'
+zip_file = 'https://automl-mm-bench.s3.amazonaws.com/ner/multimodal_ner.zip'
+from autogluon.core.utils.loaders import load_zip
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+```
+
+Next, we will load the CSV files.
+
+```{.python .input}
+dataset_path = download_dir + '/multimodal_ner'
+train_data = pd.read_csv(f'{dataset_path}/twitter17_train.csv')
+test_data = pd.read_csv(f'{dataset_path}/twitter17_test.csv')
+label_col = 'entity_annotations'
+```
+
+We need to expand the image paths to load them in training.
```

Comment:
If this is necessary for many NER datasets, do we need to add it as a util function?

Reply:
This is only for tutorial purpose, borrowed from this [tutorial]( http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-2556/9ffb8b9/tutorials/multimodal/multimodal_prediction/beginner_multimodal.html).